WalletConnect/WCInteractor.swift

```diff
@@ -16,6 +16,7 @@ public typealias BnbSignClosure = (_ id: Int64, _ order: WCBinanceOrder) -> Void
 public typealias CustomRequestClosure = (_ id: Int64, _ request: [String: Any]) -> Void
 public typealias ErrorClosure = (Error) -> Void
 public typealias TransactionSignClosure = (_ id: Int64, _ transaction: WCTrustTransaction) -> Void
+public typealias TrustGetAccountsClosure = (_ id: Int64) -> Void
```

Comment:
let's use `RequestAccounts`?

Reply:
I and Viktor discussed about this and we decided to use `getAccounts`. @vikmeup what do you think?